yt/yt/tests/integration/master/test_node_tracker.py

```diff
@@ -449,3 +449,49 @@ def get_online_node_count():
         wait(lambda: get_online_node_count() == 1)
         sleep(5)
         assert get_online_node_count() == 1
+
+
+################################################################################
+
+
+class TestRackDataCenter(YTEnvSetup):
+    NUM_MASTERS = 1
+    NUM_NODES = 1
+    NUM_SCHEDULERS = 1
+    rack_set = "testracktest"
+    data_center_set = "testdctest"
+    DELTA_NODE_CONFIG = {
+         "rack": rack_set,
+         "data_center": data_center_set,
+    }
+
+    @authors("gritukan")
```

Comment:
Not sure whether here be gritukan; it's either you or (if the maintainers would like to) the reviewer from the maintainers.